backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/theme/BpkTheme.kt

```diff
@@ -37,34 +47,75 @@ fun BpkTheme(
   content: @Composable () -> Unit,
 ) {
   val typography = BpkTypography(defaultFontFamily = fontFamily)
+  val colors = if (isSystemInDarkTheme()) BpkColors.dark() else BpkColors.light()
+
   CompositionLocalProvider(
     LocalBpkTypography provides typography,
+    LocalBpkColors provides colors,
   ) {
-    val materialTypography = Typography(
-      defaultFontFamily = fontFamily,
-      h1 = typography.hero2,
-      h2 = typography.hero4,
-      h3 = typography.hero5,
-      h4 = typography.heading2,
-      h5 = typography.heading3,
-      h6 = typography.heading4,
-      subtitle1 = typography.bodyDefault,
-      subtitle2 = typography.footnote,
-      body1 = typography.bodyDefault,
-      body2 = typography.footnote,
-      button = typography.label2,
-      caption = typography.caption,
-      overline = typography.caption.copy(fontWeight = FontWeight.Bold),
-    )
     MaterialTheme(
-      typography = materialTypography,
+      typography = typography.toMaterialTypography(fontFamily),
+      colors = colors.toMaterialColors(),
+      shapes = bpkShapes(),
       content = content,
     )
   }
 }
 
 object BpkTheme {
+
   val typography: BpkTypography
     @Composable
     get() = LocalBpkTypography.current
+
+  val colors: BpkColors
+    @Composable
+    get() = LocalBpkColors.current
+
+  val shapes: Shapes
```

Comment:
Shapes is here to allow customers to write code wthout refering to MaterialTheme